packages/wallet-connect-sdk-svelte/README.md

```diff
@@ -0,0 +1,51 @@
+# WalletConnect SDK Svelte
+
+- [Installation](#installation)
+- [Setup](#setup)
+- [Usage](#usage)
+
+## Installation
+
+Install the dependencies on your application
+
+### NPM
+
+```
+npm i @cityofzion/wallet-connect-sdk-svelte @walletconnect/sign-client @walletconnect/types
+```
+
+### YARN
+
+```
+yarn add @cityofzion/wallet-connect-sdk-svelte @walletconnect/sign-client @walletconnect/types
+```
+
+## Setup
+To begin development, first establish an account on the [Wallet Connect website](https://walletconnect.com/). Next,
+generate a new Project. This process is straightforward, requiring only a few form fields.
+
+### Instantiate an object of the WCSDKStore class
+
+```ts
+import { WCSDKStore } from '@cityofzion/wallet-connect-sdk-svelte'
+
+const wcOptions = {
+    projectId: '<your wc project id>', // the ID of your project on Wallet Connect website
+    relayUrl: 'wss://relay.walletconnect.com', // we are using walletconnect's official relay server
+    metadata: {
+        name: 'MyApplicationName', // your application name to be displayed on the wallet
+        description: 'My Application description', // description to be shown on the wallet
+        url: 'https://myapplicationdescription.app/', // url to be linked on the wallet
+        icons: ['https://myapplicationdescription.app/myappicon.png'] // icon to be shown on the wallet
+    }
+};
+
+<script>
+    const wcSdk = new WCSDKStore(wcOptions)
+</script>
+```
+
+`autoManageSession` will reload the user's connected session and subscribe to the `disconnect` event. If you don't want this at startup, set the second constructor parameter to false and call the `manageSession` method at the correct time.
```

Comment:
There is no `autoManageSession` on Svelte package. So you can remove this line.
You will need to add the documentation about [manageSession](https://github.com/CityOfZion/wallet-connect-sdk/blob/main/packages/wallet-connect-sdk-core/README.md#manage-session).

Reply:
I added autoManageSession to the SDK to be the same as react
![Screenshot 2023-10-27 at 15 45 16](https://github.com/CityOfZion/wallet-connect-sdk/assets/57366068/9fdc516f-f725-4d86-b3c5-dc1e6f300c9e)
I will add the information in read me too